pkg/virt-launcher/virtwrap/agent-poller/agent_parser_test.go

```diff
@@ -0,0 +1,376 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
```

Comment:
2020

Reply:
Please review per commit and not per PR.
This is a moved file.

Reply:
this what i did, 
its a new file, which has part of an old file (2018),
at first i thought since its a new file it should be 2018-2020 or 2020 but if it should reserve the original time, np